library/Rules/VideoUrl.php

```diff
@@ -14,8 +14,18 @@
 namespace Respect\Validation\Rules;
 
 use Respect\Validation\Exceptions\ComponentException;
+use function mb_strtolower;
+use function preg_match;
+use function sprintf;
 
-class VideoUrl extends AbstractRule
+/**
+ * Validates if the input is a video URL value:
+ *
+ * @author Danilo Correa <danilosilva87@gmail.com>
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ * @author Ricardo Gobbo <ricardo@clicknow.com.br>
+ */
+final class VideoUrl extends AbstractRule
```

Comment:
I don't think the `$service` attribute is used anywhere. Could you drop it?

Reply:
I also think, but when I delete this attribute the integration tests have failed, it is referenced in the exceptions.